VideoWeb/VideoWeb.AcceptanceTests/Steps/BrowserSteps.cs

```diff
@@ -98,7 +98,17 @@ private UserDto GetDefaultParticipant()
         private UserDto GetMatchingDisplayName(string userType)
         {
             if (_c.Test.Users.Count != 0)
-                return Users.GetUserFromDisplayName(_c.Test.Users, userType.Replace(" ", string.Empty));
+                return Users.GetUserFromUserType(_c.Test.Users, userType.Replace(" ", string.Empty));
```

Comment:
GetMatchingDisplayName and GetMatchingUserType have the same body, is this intended? Shouldn't GetMatchingDisplayName call something like GetUserFromDisplayName?

Reply:
That's corrected, an error when copying code and revising.